zeton/templates/index_child.html

```diff
@@ -68,15 +70,14 @@ <h4>NAGRODY</h4>
     <div style="clear:both"></div>
     <div id="bany_warny" class="square">
         <h2 style="text-align: center;"> *****Warny i bany****** </h2>
```

Comment:
Może się czepiam, ale nazwa *****Warny i bany****** mi nie pasuje. Mamy jeszcze KICK'a. 
Więc albo:
 *****Warny, kick i bany******
albo ogólnie
 *****Warny******

Reply:
Nazwy tej skecji jeszcze do końca nie wymyśłiłem. Dlatego jest ona w gwiazdkach. Może ktoś ma jakiś pomysł? Prawidłowo powinno być "Konsekwencje", ale dla młodego człowieka lepiej jest dostać warna lub bana niż konsekwencję. Myślę że to zostawmy narazie w tych gwiazdkach i wrócimy do tego tematu później.